src/pages/Board.pages.js

```diff
@@ -1,7 +1,138 @@
+import { useEffect, useState } from "react";
 import styled from "styled-components";
+import { colors } from "../styles/Theme";
 
 export const Board = () => {
-  return <BoardContainer>Board</BoardContainer>;
+  const [pageNationNumber, setPageNationNumber] = useState(1);
+
+  useEffect(() => {
+    fetch(`/data/board-data.json`)
+      .then(res => res.json())
+      .then(data => {
+        sessionStorage.setItem("data", JSON.stringify(data));
+      });
+  }, []);
+
+  const boardData = JSON.parse(sessionStorage.getItem("data")).slice(
+    0 + 5 * (pageNationNumber - 1),
+    5 + 5 * (pageNationNumber - 1)
+  );
+
+  const PageNationNumberData = Array(
+    Math.ceil(JSON.parse(sessionStorage.getItem("data")).length / 5)
+  ).fill();
+
+  return (
+    <>
+      <TopContainer>
+        <BoardTitle>게시판</BoardTitle>
+        <TextWrite>글쓰기</TextWrite>
+      </TopContainer>
+      <BoardContainer>
+        <BoardListWrapar>
```

Comment:
```suggestion
        <BoardListWrapper>
```
wrapar 에 제대로된 철자는 wrapper 입니다!